components/org.wso2.carbon.identity.recovery.endpoint/src/main/resources/api.identity.recovery.yaml

```diff
@@ -108,12 +108,12 @@ paths:
         - Password Recovery
         - Notification
             
-    # Endpoint uses to reset user password
+    # Endpoint used to reset user password
```

Comment:
Isn't this "The endpoint that used to reset user password"

Reply:
Fixed.